dm/syncer/data_validator.go

```diff
@@ -1309,6 +1318,29 @@ func (v *DataValidator) OperateValidatorError(validateOp pb.ValidationErrOp, err
 	return v.persistHelper.operateError(tctx, toDB, validateOp, errID, isAll)
 }
 
+func (v *DataValidator) UpdateValidator(req *pb.UpdateValidationWorkerRequest) error {
+	var (
+		pos *mysql.Position
+		gs  mysql.GTIDSet
+		err error
+	)
+	if len(req.BinlogPos) > 0 {
+		pos, err = binlog.VerifyBinlogPos(req.BinlogGTID)
```

Comment:
```suggestion
		pos, err = binlog.VerifyBinlogPos(req.BinlogPos)
```

better to add basic test about this feature